yarn.lock

```diff
@@ -2617,6 +2617,11 @@ prettier@^2.7.1:
   resolved "https://registry.yarnpkg.com/prettier/-/prettier-2.8.8.tgz#e8c5d7e98a4305ffe3de2e1fc4aca1a71c28b1da"
   integrity sha512-tdN8qQGvNjw4CHbY+XXk0JgCXn9QiF21a55rBe5LJAU+kDyC4WQn4+awm2Xfk2lQMk5fKup9XgzTZtGkjBdP9Q==
 
+prettier@^3.3.3:
```

Comment:
Here can use yarnpkg.com npm source

Reply:
Can you tell me how to do it? I executed `yarn add prettier -D`

Reply:
I changed it from 'https://registry.npmmirror.com/' to 'https://registry.yarnpkg.com/'. Please review again. Thanks.